generated/thrift/rpc.thrift

```diff
@@ -107,7 +108,8 @@ struct BlocksMetadata {
 struct BlockMetadata {
 	1: required i64 start
 	2: optional i64 size
-	3: optional Error err
+	3: optional i64 checksum
+	4: optional Error err
```

Comment:
Hm, maybe we should make err as field 1 so we don't have to make breaking changes in the future like this?